tie2misp

```diff
@@ -24,15 +26,43 @@ from loader import Loader
                                                'file.')
 @click.option('--date', help='If a date is set, instead of using the actual date the parser will use the '
                              'given date. The date must be given in the following format YYYY-MM-DD.')
-def startup(type, delay, file, noupload, date):
-    error = False
-    given_date = ''
+@click.option('--loglvl', default=20, help='Sets the log level. Default is 20.\n Params are: 0 - NOTSET / 10 - DEBUG / '
+                                           '20 - INFO / 30 - WARNING / 40 - ERROR / 50 - CRITICAL')
+@click.option('--disable_console_log', is_flag=True, help='If used, the parse will not generate output in the console')
+@click.option('--disable_file_log', is_flag=True, help='If used, the parser will not generate output in the console')
+@click.option('--severity', type=int, help='Events with a severity value lower than the given value will not be parsed. If used, '
+                                 'the parser will ignore the severity value defined in the config file. The severity '
+                                 'value can be equal or between 0 and 5.')
+@click.option('--confidence', type=int, help='Events with a confidence value lower than the given value will not be parsed. If '
+                                   'used, the parser will ignore the severity value defined in the config file. The '
+                                   'confidence value can be equal or between 0 and 100')
+def startup(type, delay, file, noupload, date, loglvl, disable_console_log, disable_file_log, severity, confidence):
     """
     Starting the Parser with a given type of events that should be created.
```

Comment:
IMHO 'converter' and 'convert' are better throughout the documentation instead of 'parser' and 'parse'. The tool does more than just parse the input.